### PR TITLE
[cloud_infra_center] Remove the force update dns and update Doc

### PR DESCRIPTION
**Pull request by shhull**
1. Remove the dns force update for subnet
2. Add ICIC 1.1.6 support
3. Update ReadME for subnet DNS

### CODE REVIEW

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The playbook contains the following topics:
 
   3. Requirements pre-check before the installation
 
-**Note**: This playbook supports IBM® Cloud Infrastructure Center version 1.1.4, 1.1.5 and RH OpenShift Container Platform version 4.6 and, 4.7, 4.8, 4.9, 4.10, 4.11for z/VM and version 4.7, 4.8, 4.9, 4.10, 4.11for KVM.
+**Note**: This playbook supports IBM® Cloud Infrastructure Center version 1.1.4, 1.1.5, 1.1.6 and RH OpenShift Container Platform version 4.6 and, 4.7, 4.8, 4.9, 4.10, 4.11 for z/VM and version 4.7, 4.8, 4.9, 4.10, 4.11 for KVM.
```
Comment (zhengxiaomei123) [inline z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:37]: Currently we support ocp 4.10, 4.11 and 4.12. Please help update it.

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md [hunk 2]
```diff
@@ -224,8 +223,25 @@ icic-services status
 ```
 - Login to the IBM Cloud Infrastructure Center web console and select Home > Environment Checker, click the Run Environment Checker button to confirm the cluster does not have any failed messages.
 
-If you meet any **not running** service or **failed** message, check the IBM Cloud Infrastructure Center [Troubleshooting](https://www.ibm.com/docs/en/cic/1.1.5?topic=troubleshooting) document to fix before running the ansible playbook.
+If you meet any **not running** service or **failed** message, check the IBM Cloud Infrastructure Center [Troubleshooting](https://www.ibm.com/docs/en/cic/1.1.6?topic=troubleshooting) document to fix before running the ansible playbook.
 
+9. Subnet DNS (Required for KVM)
+
+**Note**: This step is required for KVM and only necessary if you want to use your own or external bastion server resolvers your Nova servers, z/VM is optional.
```
Comment (zhengxiaomei123) [inline z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:230]: `only necessary if you want to use your own or external bastion server resolvers your Nova servers,`
I think this description is indigestible. Just delete it. 
This step is required for KVM , z/VM is optional.

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md [hunk 3]
```diff
@@ -80,11 +80,10 @@ After you performed the previous steps successfully, you get one ready OpenShift
...
       - Disk with at least 20 GiB
-- **(Optional)** A Bastion server, a machine that is used to configure DNS and Load Balancer for the Red Hat OpenShift installation
-    - If you use your own existing DNS server and Load Balancer for the Red Hat OpenShift installation, the bastion server **is not** required.
-    - If you have external or existing DNS server, but no Load Balancer for the Red Hat Openshift installation, please set `os_dns_domain` property, and then use a separate YAML `configure-haproxy` to configure the HAProxy in bastion server.
-    - If you have existing Load Balancer, but no DNS server for the Red Hat Openshift installation, you can use a separate YAML `configure-dns` to configure the DNS server in bastion server.
-    - If you don't have any existing DNS server or Load Balancer for the Red Hat Openshift installation, you need to create one Linux server as the bastion server and run playbook to configure DNS server and Load Balancer. You can also use the same Linux server that runs Ansible.
+- **(Optional)** A Bastion server, a machine that is used for deploying and managing the OpenShift container platform cluster
```
Comment (zhengxiaomei123) [inline z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:83]: Why we change this part? The bastion server is still the machine used to configure DNS and Load Balancer for the Red Hat OpenShift installation. And use still has 3 choices to install their DNS and Load Balancer service. What the difference is user need to set the subnet DNS record as the DNS server. Along to the 3 choices, the DNS server can be bastion server or user's external DNS server.

**Review by zhengxiaomei123 — CHANGES_REQUESTED** [review]

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md (hunk 3 above)
Comment (shhull) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:83]: not remove it, just combine the first and second item. And leave two choices here in preparation, user can get details about DNS and LB in Step2.
```
- **(Optional)** A Bastion server, a machine that is used for deploying and managing the OpenShift container platform cluster
    - This server is required if you want to use your own bastion server to resolve your Nova servers.
      - You can use our playbook to automatic config the DNS and Load Balancer, check [Step2](#creation-of-the-cluster).
    - This server is optional if you could add DNS and Load Balancer into your external bastion server manually, you can refer [Add-DNS-HAProxy](docs/add-dns-haproxy.md) to update it.
    - The firewalld service should be enabled and running in bastion server.
```

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md (hunk 1 above)
Comment (shhull) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:37]: done.

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md (hunk 2 above)
Comment (shhull) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:230]: done.

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ The playbook contains the following topics:
 
   3. Requirements pre-check before the installation
 
-**Note**: This playbook supports IBM® Cloud Infrastructure Center version 1.1.4, 1.1.5 and RH OpenShift Container Platform version 4.6 and, 4.7, 4.8, 4.9, 4.10, 4.11for z/VM and version 4.7, 4.8, 4.9, 4.10, 4.11for KVM.
+**Note**: This playbook supports IBM® Cloud Infrastructure Center version 1.1.4, 1.1.5, 1.1.6 and RH OpenShift Container Platform version 4.6 and, 4.7, 4.8, 4.9, 4.10, 4.11, 4.12 for z/VM and version 4.7, 4.8, 4.9, 4.10, 4.11, 4.12 for KVM.
```
Comment (zhengxiaomei123) [inline z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:37]: we only support 4.10, 4.11 and 4.12 .

**Review by zhengxiaomei123 — CHANGES_REQUESTED** [review]

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md (hunk 4 above)
Comment (shhull) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:37]: update it. 
**Note**: This playbook supports IBM® Cloud Infrastructure Center version 1.1.5, 1.1.6 and RH OpenShift Container Platform version 4.10 and 4.11, 4.12 for z/VM and version 4.10, 4.11, 4.12 for KVM.

**Pull request merged**